e2e/testresources/elasticsearch-legacy-monitoring.json

```diff
@@ -0,0 +1,1344 @@
+{
```

Comment:
These JSON files are extracted from here:
- https://internal-ci.elastic.co/job/elastic+estf-monitoring-snapshots+7.7+multijob-beat/50/consoleText
- https://internal-ci.elastic.co/job/elastic+estf-monitoring-snapshots+7.7+multijob-elasticsearch/50/consoleText
- https://internal-ci.elastic.co/job/elastic+estf-monitoring-snapshots+7.7+multijob-kibana/50/consoleText

And they are very useful for debugging the helper functions to handle special cases.
- https://internal-ci.elastic.co/job/elastic+estf-monitoring-snapshots+7.7+multijob-logstash/50/consoleText